src/CloudCode.js

```diff
@@ -16,19 +18,19 @@
  *
  * If you want to use afterDelete for a predefined class in the Parse JavaScript SDK (e.g. {@link Parse.User}), you should pass the class itself and not the String for arg1.
  * ```
- * Parse.Cloud.afterDelete('MyCustomClass', (request) => {
+ * Parse.Cloud.afterDelete('MyCustomClass', async (request) => {
```

Comment:
i just curios about this CloudCode. Is it work in browser now? or is your long-team plan?

Reply:
No it's in the server, but that's easier to have the whole cloud code API documentation with the rest of the JS SDK, we may move it at a later point if we generate a bigger API docs for parse-server.

Reply:
okay, i see i just confused :)